hudi-common/src/test/java/org/apache/hudi/common/table/read/buffer/TestKeyBasedFileGroupRecordBuffer.java

```diff
@@ -176,12 +205,49 @@ void readWithCommitTimeOrdering() throws IOException {
     assertEquals(2, readStats.getNumUpdates());
   }
 
+  @Test
+  void readWithCommitTimeOrderingWithRecords() throws IOException {
```

Comment:
Could this test and other new tests for streaming loader be removed since it's added in the new test class for streaming record buffer loader?